src/experimental/patterns/__snapshots__/kinesis-lambda.test.ts.snap

```diff
@@ -259,16 +259,6 @@ exports[`The GuKinesisLambda pattern should create the correct resources for a n
                 ],
               },
             },
-            {
-              "Action": "kinesis:DescribeStream",
-              "Effect": "Allow",
-              "Resource": {
-                "Fn::GetAtt": [
-                  "KinesisStream46752A3E",
-                  "Arn",
-                ],
-              },
-            },
```

Comment:
AWS CDK appear to have dropped this permission, which was added via [`KinesisEventSource`](https://github.com/guardian/cdk/blob/0baddafe92e97f4b5a4007b1772affe37bbcf7e5/src/experimental/patterns/kinesis-lambda.ts#L115).